docs/alerting/api.md

```diff
@@ -858,3 +858,281 @@ _opendistro/_alerting/destinations/<destination-id>
   "_primary_term": 1
 }
 ```
+---
+
+## Create email account
+
+#### Request
+```json
+POST _opendistro/_alerting/destinations/email_accounts
+{
+  "name": "example_account",
+  "email": "example@email.com",
+  "host": "smtp.email.com",
+  "port": 465,
+  "method": "ssl"
+}
+```
+
+#### Sample response
+```json
+{
+  "_id" : "RAau5XQBBF7ehDf33O3G",
+  "_version" : 1,
+  "_seq_no" : 7,
+  "_primary_term" : 2,
+  "email_account" : {
+    "schema_version" : 2,
+    "name" : "example_account",
+    "email" : "example@email.com",
+    "host" : "smtp.email.com",
+    "port" : 465,
+    "method" : "ssl"
+  }
+}
+```
+
+## Update email account
+
+#### Request
+```
+PUT _opendistro/_alerting/destinations/email_accounts/<email_account_id>
+``` 
+
+## Get email account
+
+#### Request
+```
+GET _opendistro/_alerting/destinations/email_accounts/<email_account_id>
+```
+
+## Delete email account
+
+#### Request
+```
+DELETE _opendistro/_alerting/destinations/email_accounts/<email_account_id>
+```
+#### Sample response
+
+```json
+{
+  "_index" : ".opendistro-alerting-config",
+  "_type" : "_doc",
+  "_id" : "<email_account_id>",
+  "_version" : 1,
+  "result" : "not_found",
```

Comment:
This should be:

```
"result" : "deleted"
```

if the document being deleted exists. Everything else in the response looks fine though.

Reply:
done